backend/common/src/main/java/ai/verta/modeldb/common/config/Config.java

```diff
@@ -129,7 +128,11 @@ public FutureJdbi initializeJdbi(DatabaseConfig databaseConfig, String poolName)
     hikariDataSource.setMetricsTrackerFactory(new PrometheusMetricsTrackerFactory());
     hikariDataSource.setPoolName(poolName);
 
-    final Jdbi jdbi = Jdbi.create(hikariDataSource);
+    if (enableTrace) {
+      final var tracer = Configuration.fromEnv().getTracer();
```

Comment:
I _think_ we have to use the same tracer, vs creating a new one. I don't know if this creates a new one or not.

Reply:
I think it must create a new one, because the spans aren't linked.

Reply:
I'll research how to use the existing tracer.